ecosystem/sep-0024.md

```diff
@@ -818,7 +906,9 @@ The response should be a JSON object like:
   },
   "features": {
     "account_creation": true,
-    "claimable_balances": true
+    "claimable_balances": true,
+    "indicative_quotes_supported": true,
+    "firm_quotes_supported": true
```

Comment:
> Wallets can determine if the 1st approach is possible by looking at the toml, so I don't think the flag in GET /info is necessary for that.

I agree that they can simply look at the toml file. IMO, it's a nice addition to communicate to the wallets what features anchor support

> I'm not sure if that information is useful

It's mainly to tell if `POST /quotes` endpoint is supported or not. AFAIK, all current anchors support both endpoints, so it's an extra addition to make it more flexible and understandable for the wallet.

That being said, I don't feel strongly about adding this flags. We can remove them, if majority or reviewers think it's not necessary.

Reply:
I don't have a strong opinion here either. I can think that this could help a bit to automate feature enabling/disabling by Anchors (e.g. an Anchor that is connected to several wallets but needs to turn quote off only to a specific wallet without having to change the TOML file), but I'm leaning towards agreeing that it's not a necessary change, I have a feeling those flags would be rarely used

Reply:
That's a valid concern. I think I'll remove them in the final version of the PR.

Reply:
Coming back to this, I think I agree with cassio that these flags would not be used in practice. I think it is a valid assumption that if `ANCHOR_QUOTE_SERVER` is present, firm quotes are supported in SEP-24 transactions.

Reply:
What if anchor supports both SEP-31 and SEP-24, but doesn't support SEP-38 for SEP-24? Do we want to support such case or leave it to the anchor? (They can throw an error if context set to SEP-24)

Reply:
I feel fairly confident making the assumption that the anchor would want to enable the use of SEP-38 quotes in all the services that support it.